sound/soc/sof/ipc.c

```diff
@@ -667,12 +814,13 @@ int snd_sof_ipc_set_get_comp_data(struct snd_sof_ipc *ipc,
 {
 	struct snd_sof_dev *sdev = ipc->sdev;
 	struct sof_ipc_ctrl_data *cdata = scontrol->control_data;
+	struct sof_ctrl_data_send_params sipc;
 	int send;
 	int err = 0;
 
 	send = sof_get_ctrl_ipc_dir(ctrl_type);
 
-	/* read or write firmware volume */
+	/* read or write firmware volume fast */
```

Comment:
not sure I get why the comment is changed? You are saying too much or too little here.

Reply:
I'll change it back. I was trying to preserve the old code, which I actually don't have too much clue what it is doing and why it is there. As the old comment said the later ipc is "slow" I guess this whatever method coming first is then "fast". but yeah it is old code which I really can't comment very well.